### PR TITLE
Update MqlServerUrl query to use the preferred approach in the core API

### PR DESCRIPTION
**Pull request by erikvorhes**
# Changes
* See title
* Also fixes a typo
* No version bump because v1.16.0 has not been published yet

_Manual changes are to the following files (i.e., everything else is generated):_
* context/MqlContext/MqlContextProvider.tsx
* queries/core/MqlServerUrlQuery.ts 

# Security Implications
None

### CODE REVIEW

**Review by conglei — APPROVED** [review]
LGTM

File: context/MqlContext/MqlContextProvider.tsx [hunk 1]
```diff
@@ -139,11 +139,9 @@ function MqlContextProviderInternal({
     handleCombinedError,
   });
 
-
-
   useEffect(() => {
     const stateToUpdate: Partial<MqlContextType> = {};
-    const useOverride = !!mqlServerUrlOverride;
+    const useOverride = Boolean(mqlServerUrlOverride);
```
Comment (marcodamore) [inline context/MqlContext/MqlContextProvider.tsx:144]: It seems to me that we don't really need all this logic about overrides. Hopefully the backend API abstracts this logic away. Though that can probably wait until we have more time to rip it out

**Review by marcodamore — APPROVED** [review]

File: context/MqlContext/MqlContextProvider.tsx (hunk 1 above)
Comment (erikvorhes) [reply context/MqlContext/MqlContextProvider.tsx:144]: Yeah. In this case I believe it's to pass in a value via an environment variable, if you want to override it without hitting the backend.

Comment (erikvorhes) [reply context/MqlContext/MqlContextProvider.tsx:144]: It does feel needlessly complex, though!

Comment (erikvorhes) [reply context/MqlContext/MqlContextProvider.tsx:144]: This is def. more complex than it needs to be. I may revisit this today before deploying a new version of the package.

**Pull request merged**